vnext/Mso/motifCpp/motifCppTest.h

```diff
@@ -238,6 +238,10 @@ struct TerminateHandlerRestorer {
   std::terminate_handler Handler;
 };
 
+#pragma warning(push)
+#pragma warning(disable : 4611) // interaction between '_setjmp' and C++ object destruction is non-portable
```

Comment:
probably not, it's fine from and SDL standpoint for warnings to exist in test code, though it may be good to fix them if they impact test reliability etc. Main thing is to ensure product code is and remains sdl warning clean

Reply:
This is test-only code and using setjmp is essential for this function. There is no way to address it in this function differently except for deleting it which will stop us from testing important scenarios where we want to verify that code terminates under tested conditions.
Thus, the best way is to disable this warning to be able to continue testing these scenarios.

Reply:
Good point! I will add a condition in the project file to ensure it.

Reply:
Added the new MSO_MOTIFCPP macro to ensure that the motifCpp Mso liblet can be used only in the unit test projects.
This is a quick fix for this PR. I am going to implement a more elaborate system of Mso liblet dependencies as a part of a different PR.